util/logger.js

```diff
@@ -8,21 +11,35 @@ exports.notifyError = function (error, config, subject) {
     // Log error to CLI
     log.error('mongomonitor', new Date(), error.message);
 
-    // Generate e-mail using mailgen
-    var mail = mailer.generate({
-        body: {
-            title: subject,
-            intro: error.message,
-            outro: 'We thank you for choosing mongomonitor.'
-        }
-    });
+    // Email the error of smt settings present
+    if (config.smtp !== undefined) {
+        // Generate e-mail using mailgen
+        var mail = mailer.generate({
+            body: {
+                title: subject,
+                intro: error.message,
+                outro: 'We thank you for choosing mongomonitor.'
+            }
+        });
+
+        // Send mail via mailer
+        mailer.sendMail({
+            from: config.smtp.address,
+            to: config.smtp.address,
+            subject: `[mongomonitor] ${subject}`,
+            html: mail.html,
+            text: mail.text
+        }, config);
+    }
 
-    // Send mail via mailer
-    mailer.sendMail({
-        from: config.smtp.address,
-        to: config.smtp.address,
-        subject: `[mongomonitor] ${subject}`,
-        html: mail.html,
-        text: mail.text
-    }, config);
+    // Send Slack message if webhook settings present
+    if (config.slack !== undefined) {
+        if (slack === undefined)
+            slack = new Slack(config.slack);
```

Comment:
Please add `{` and `}` brackets to the `if` statement to conform with the rest of the project.